bin/conflict-checker.py

```
@@ -0,0 +1,239 @@
+
+    if args.stack is not None:
+        stack = _parse_stack_arg(args.stack)
+    elif args.stack_file is not None:
+        stack = _parse_stack_file(args.stack_file)
+    else:
+        stack = _parse_rule011(args.rule011)
+
+    if not args.conflicts.is_file():
+        raise SystemExit(
+            f"conflict-checker: conflicts file not found: {args.conflicts}"
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)